### DISCUSSION TITLE
Dot notion + Partial reload of page props

### DISCUSSION THREAD

**Discussion by godismyjudge95** [inertiajs/inertia#1249] [Ideas]
I was building a feature out the other day and realized it might be handy to be able to trigger an Inertia visit with the "only "option set to a dot notion key.

For example:
```js
Inertia.patch(
    route('tasks.status.update', task.id),
    {
        status: newStatus,
    },
    {
        preserveScroll: true,
        preserveState: true,
        only: ['task.status', 'flash'],
    },
);
```

The result of this would be that only the "status" key on the "task" prop would be updated.

I have a really basic MVP here:
- https://github.com/godismyjudge95/inertia-laravel/commit/c3bc44ca88080b3570b3923ecc7b3523dc6d15d5
- https://github.com/godismyjudge95/inertia/commit/85dbe9ddcd5449f0fa459b7ab935534eac796b1a

This currently works, albeit breaks the reactivity of the page props (not an issue if one is using the `usePage()` composable like the docs recommend)

Let me know what you think, I'll clean it up and submit a pull request.

**Comment by Bosphoramus** [comment]
Cool!
